styleguide/source/_patterns/00-atoms/02-text/14-caption/caption.twig

```diff
@@ -0,0 +1 @@
+<p class="caption {{ class }}">{{ content|default('Bacon ipsum dolor amet minim biltong culpa mollit brisket labore porchetta. Turkey porchetta magna, do doner burgdoggen aute excepteur nisi sint pancetta ipsum ea. Frankfurter burgdoggen pastrami eiusmod in.') }}</p>
```

Comment:
Suggestion to convert to bacon ipsum across styleguide, or use lorem ipsum here for consistency.